lib/timodule.js

```diff
@@ -739,13 +739,14 @@ async function detectPlatformSpecificNativeModuleInNPMPackage(directory, platfor
 	if (await fs.exists(manifestFile)) {
 		manifest = await readManifest(manifestFile);
 	}
-	const platformMetadadata = (json.titanium.platform && json.titanium.platform[platformName]) || {};
+	const platformMetadata = (json.titanium.platform && json.titanium.platform[platformName]) || {};
 	manifest.name = json.titanium.name || manifest.name;
```

Comment:
I think we need to try and be as consistent as possible for all properties here so:
```suggestion
	manifest.name = platformMetadata.name || json.titanium.name || manifest.name || json.name;
```

Reply:
This will work except in the cases where it is relying on package.name and they are using scoped packages.  Maybe we remove scope like your suggestion below?